Profile list should not return error if one cluster is not ready

As seen in this test

https://storage.googleapis.com/minikube-builds/logs/6845/d6e4143/VirtualBox_Linux.html#fail_TestFunctional%2fparallel%2fProfileCmd%2fprofile_list


```
--- FAIL: TestFunctional/parallel/ProfileCmd/profile_list (0.21s)
functional_test.go:540: (dbg) Run:  out/minikube-linux-amd64 profile list
functional_test.go:540: (dbg) Non-zero exit: out/minikube-linux-amd64 profile list: exit status 78 (208.745917ms)
-- stdout --
|----------------------------------------------|------------|------------|----------------|------|----------|---------|
|                   Profile                    | VM Driver  |  Runtime   |       IP       | Port | Version  | Status  |
|----------------------------------------------|------------|------------|----------------|------|----------|---------|
| containerd-20200301T065612.372950053-30423   | virtualbox | containerd |                | 8444 | v1.17.3  |         |
| crio-20200301T065612.372967901-30423         | virtualbox | crio       |                | 8443 | v1.15.7  |         |
| docker-flags-20200301T065612.372493601-30423 | virtualbox | docker     |                | 8443 | v1.17.3  |         |
| functional-20200301T065430.272841444-30423   | virtualbox | docker     | 192.168.99.250 | 8443 | v1.17.3  | Running |
| newest-cni-20200301T065612.373103701-30423   | virtualbox | docker     |                | 8443 | v1.17.3  |         |
| old-docker-20200301T065612.372806255-30423   | virtualbox | docker     |                | 8443 | v1.11.10 |         |
|----------------------------------------------|------------|------------|----------------|------|----------|---------|
-- /stdout --
** stderr ** 
X error loading profiles: state: machine does not exist
** /stderr **
functional_test.go:542: [out/minikube-linux-amd64 profile list] failed: exit status 78
```

--- a/cmd/minikube/cmd/config/profile_list.go
+++ b/cmd/minikube/cmd/config/profile_list.go
@@ -103,7 +103,7 @@ var printProfilesTable = func() {
 	}
 
 	if err != nil {
-		exit.WithCodeT(exit.Config, fmt.Sprintf("error loading profiles: %v", err))
+		glog.Warningf("error loading profiles: %v", err)
 	}
 
 }

